data.json

```diff
@@ -1007,6 +1007,15 @@
     "dateAdded": "2023-10-21T23:43:00.000Z",
     "tags": ["stickers", "clothing", "hacktoberfest"]
   },
+  {
+    "name": "WireMock",
+    "difficulty": "medium",
+    "description": "Contribute to WireMock Ecosystem projects marked with a hacktoberfest topic by their maintainers.",
```

Comment:
And get what?

Reply:
i added what type of swag you will receive if you contribute to WireMock